client/src/App.js

```diff
@@ -14,11 +13,10 @@ const App = () => {
     <Router>
       <Header />
       <Switch>
-        <Route exact path="/home" component={Home} />
+        <Route exact path="/cat" component={CategoryPage} />
         <Route exact path="/">
-          <Redirect to="/home" />
+          <Redirect to="/cat" />
```

Comment:
Can we change this from /cat to just / ? I think that it would be more professional and look better.

Reply:
Done.